src/Braille/BrailleTable.css

```diff
@@ -0,0 +1,14 @@
+.BrailleTable .table-responsive {
+    border-top: none;
+  }
+    
+  .BrailleTable th {
+    text-align: center;
+  }
+
+  .BrailleTable td {
+    text-align: center;
+    font-size: 40px;
+    font-weight: bold;
+  }

```

Comment:
lol, the odd indent again

Reply:
I blame VSCode this time.  It does something weird when copying and pasting.